content/articles/how-to-embed-compose-desktop-application-to-swing-based-application/index.md

```diff
@@ -0,0 +1,236 @@
+
+Let's have a look at when you can use `ComposePanel` or `SwingPanel` in your application.
+
+#### ComposePanel use cases
+- Whenever you would like to add an engaging rendering section to your app. This will be more simple and efficient to do with Compose.
+- Whenever you wish to include animated objects, or a grid of dynamic objects, in your app. For instance, consider a toolbar with an animated emoticon reaction to events.
+- Whenever you need to incorporate a complex display region, Compose is the way to go.
+- Whenever you need to make a complex component of a Swing-based app easier to understand, go for Compose.
+
+#### SwingPanel use cases
+- When your app does not have complexity in layouts, menus, or other features like tooltips. 
+- When adjusting the position of the Swing component, `SwingPanel` decreases the possibility of problems. This is because the position of a `SwingPanel` does not shift. 
+- When the component you wish to utilize is only accessible in Swing and not in Compose 
+
+### Using a ComposePanel in a Swing application
+Let's have an example of how our code should look like when using `ComposePanel`.
+```kotlin
+fun main() = SwingUtilities.invokeLater {
+    val jFrame = JFrame()
+    setDefaultLookAndFeelDecorated(true)
+    val panel = ComposePanel()
+    jFrame.defaultCloseOperation = WindowConstants.EXIT_ON_CLOSE
+    jFrame.title = "Swings App"
+    jFrame.isVisible = true
+    jFrame.setSize(600, 600)
+
+    jFrame.contentPane.add(panel, BorderLayout.CENTER)
+    jFrame.contentPane.add(JButton("Swing Button 1"), BorderLayout.NORTH)
+    jFrame.contentPane.add(JButton("Swing Button 2"), BorderLayout.EAST)
+    jFrame.contentPane.add(JButton("Swing Button 3"), BorderLayout.WEST)
+
+    panel.setContent { ComposeUI() }
+}
+@Composable
+fun ComposeUI (){
+    Column (
+        modifier = Modifier.fillMaxSize(),
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        Spacer(modifier = Modifier.height(50.dp))
+        Row {
+            Surface(
+                color = Color(200, 200, 200),
+                modifier = Modifier.size(130.dp, 130.dp)
+            ){
+                Column {
+                    Box(
+                        modifier = Modifier.height(30.dp).fillMaxWidth(),
+                        contentAlignment = Alignment.Center
+                    ){
+                        Text("Compose Button")
+                    }
+                    Spacer(modifier = Modifier.height(50.dp))
+                    Box(
+                        contentAlignment = Alignment.Center,
+                        modifier = Modifier.height(30.dp).fillMaxWidth()
+                    ){
+                        Button(onClick = {}, shape = RoundedCornerShape(10.dp)){
+                            Text("Button 1")
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
```

Comment:
Once again, please include the import statements to make it clear which classes you are importing and from where